internal/applicationsnapshot/report.go

```diff
@@ -153,7 +153,7 @@ func (r Report) WriteAll(targets []string, p format.TargetParser) (allErrors err
 
 		if data, err := r.toFormat(target.Format); err != nil {
 			allErrors = multierror.Append(allErrors, err)
-		} else if _, err := target.Write(data); err != nil {
+		} else if _, err := target.Write(append(data, "\n"...)); err != nil {
```

Comment:
Nitpick: I think I would have made this "smart", i.e. inject a line break only if the output doesn't already end with a line break.

Also I don't understand the `...` syntax here. Is it because data is `[]byte`?